src/lib/utils/skin.ts

```diff
@@ -113,16 +125,39 @@ export function isSkin(asset: rgAsset): boolean {
 }
 
 export function isCharm(asset: rgAsset): boolean {
-    if (asset.market_hash_name.startsWith('Charm')) {
-        // Tags aren't available on SCM items, so use a MHN heuristic instead
+    return isAbstractType(asset, 'Charm', 'CSGO_Type_Charm');
+}
+
+export function isAgent(asset: rgAsset): boolean {
+    return isAbstractType(asset, 'Agent', 'Type_CustomPlayer');
+}
+
+export function isSticker(asset: rgAsset): boolean {
+    return isAbstractType(asset, 'Sticker', 'CSGO_Tool_Sticker');
+}
+
+export function isPin(asset: rgAsset): boolean {
+    return isAbstractType(asset, 'Pin', 'CSGO_Type_Collectible');
```

Comment:
I believe CSGO_Type_Collectible may encompass more than just Pins?

For example, badges like a 10 Year Veteran Coin have an inspect link and are also a collectible: https://steamcommunity.com/id/Step7750/inventory#730_2_37136727351

Reply:
You are right! Actually, I just noticed that CSFloat doesn't even support selling pins, so we can just remove this again.